js/usercss.js

```diff
@@ -0,0 +1,503 @@
+/* global loadScript mozParser semverCompare colorParser */
+'use strict';
+
+// eslint-disable-next-line no-var
+var usercss = (() => {
+  // true for global, false for private
+  const METAS = {
+    __proto__: null,
+    author: true,
+    advanced: false,
+    description: true,
+    homepageURL: false,
+    // icon: false,
+    license: false,
+    name: true,
+    namespace: false,
+    // noframes: false,
+    preprocessor: false,
+    supportURL: false,
+    'var': false,
+    version: false
+  };
+
+  const META_VARS = ['text', 'color', 'checkbox', 'select', 'dropdown', 'image'];
+
+  const BUILDER = {
+    default: {
```

Comment:
[`default`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Keywords) is a reserved word, and if used in an object it should be wrapped in quotes; but I think avoiding it is best. Change it to `defaults`.

Reply:
Wouldn't `defaultValue` be semantically better than `defaults`?

Reply:
Yes, `defaultValues` is better... I meant that to be an example.

Reply:
Er, I meant defaultValue (singular) in your second comment for `const result = {`, but here `'default'` in quotes is semantically better than `defaults` and I see no good alternative.

Reply:
Anyway, reserved words are allowed to be used as unquoted property names as of ES5 so we can leave `default` as is without quotes.